design/one-pager-stack-status-spec.md

```diff
@@ -0,0 +1,82 @@
+# Stack UI Schema Status Spec
+
+- Owner: Steven Rathbauer (@rathpc)
+- Reviewers: Crossplane Maintainers
+- Status: Draft
+
+## Proposal
+
+[crossplaneio/crossplane#929](https://github.com/crossplaneio/crossplane/issues/929)
+
+## Problem
+
+As of right now there is no defined spec to be able to drive status information for a given stack or resource from a GUI perspective. It would be useful to add keys to the existing `ui-schema.yaml` file that can be used for this purpose.
```

Comment:
This problem statement could use more background information.  It doesn't provide necessary context for a new reader to this subject to understand what is being discussed and why it is needed.  Can you please add more background and/or examples about what the design wants to accomplish?

Reply:
Will add more info